doc/build-unix.md

```diff
@@ -163,8 +163,8 @@ symbols, which reduces the executable size by about 90%.
 miniupnpc
 ---------
 
-[miniupnpc](http://miniupnp.free.fr/) may be used for UPnP port mapping.  It can be downloaded from [here](
-http://miniupnp.tuxfamily.org/files/).  UPnP support is compiled in and
+[miniupnpc](https://miniupnp.tuxfamily.org) may be used for UPnP port mapping.  It can be downloaded from [here](
```

Comment:
Could also fix double white spaces between the sentences while you are at it :)

Reply:
The white space hasn't changed with this PR, and doesn't really need fixing either. I'm just going to merge this, because it's already been bike-shed to death.